Image_process/processImage.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
+        self.image_path = image_path
+
+    def resize_image(self, width=128, height=128):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        if width <= 0 or height <= 0:
+            raise ValueError("Width and height must be positive integers.")
+        #if image isn't square raise a warning and resize to square
+        image = Image.open(self.image_path)
+        if image.size[0] != image.size[1]:
+            print("Warning: Image is not square. Resizing to square.")
+
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
+        """
+        Save the image to the specified path.
+        :param image: Image object to save.
+        :param save_path: File path to save the image.
+        """
+        
+        # Ensure the save directory exists
+        os.makedirs(os.path.dirname(save_path), exist_ok=True)
+        # Save the image
+        image.save(save_path)
+
+class GifProcess:
```

Comment:
similar to my prior comment, I don't see why this is a class instead of multiple functions.